.gitignore

```diff
@@ -3,3 +3,4 @@
 composer.lock
 phpunit.xml
 vendor/
+.idea/
```

Comment:
This has nothing to do with this bundle. I recommend to add the .idea directory to your local .git/info/exclude file

Reply:
Done (commit cd8b4a6)